pages/main/faq.md

```diff
@@ -7,7 +7,43 @@ layout: documentation
 
 This page contains some Frequently Asked Questions.
 ## What is AsteroidOS?
-AsteroidOS is an open source Linux distribution that runs on many different smartwatches. It uses [Qt](http://www.qt.io/) and QML to provide the graphical interface.
+AsteroidOS is an open source Linux distribution that runs on many different smartwatches. It uses [Qt](http://www.qt.io/) and QML to provide the graphical interface.\
+One of the main goals of AsteroidOS is to provide users with greater control over their devices and data privacy. The operating system is fully customizable, and users can modify it to meet their specific needs.
+
+## Which features does AsteroidOS provide?
+Currently, AsteroidOS has these features available:
+ - phone notifications
+ - always on display
+ - tilt to wake
+ - palm to sleep
+ - multiple launcher styles
+ - (live) wallpapers
+ - [community watchfaces](https://github.com/AsteroidOS/unofficial-watchfaces)
+ - nightstand mode
+
+## Does AsteroidOS have any apps included?
+The following apps are delivered with an AsteroidOS installation:
+ - agenda, a calender
+ - alarm clock
+ - calculator
+ - compass
+ - diamonds game
+ - flashlight
+ - heartrate check
+ - music, a MPRIS media remote control
```

Comment:
I'm not sure if mentioning "MPRIS" is relevant here.

Reply:
Agreed.  Most won't know what that or why they'd care.

Reply:
but media remote control instead of just music remote control as before checks out?
I mean, due to mpris we can control any media, not just music?